docs/resources/obs_bucket.md

```diff
@@ -254,8 +257,13 @@ The `logging` object supports the following:
 
 * `target_bucket` - (Required, String) The name of the bucket that will receive the log objects. The acl policy of the
   target bucket should be `log-delivery-write`.
+
 * `target_prefix` - (Optional, String) To specify a key prefix for log objects.
 
+* `agency` - (Optional, String) The agency must be an OBS cloud service agency and has the `PutObject` permission.
+If default encryption is enabled for the current bucket, ensure that the agency has the `KMS Administrator permissions`
+in the region where the current bucket is located.
```

Comment:
1. please add two spaces at the begin of each line.
2. suggest to use "Specifies the IAM agency of ..."

Reply:
OK